components/CheckDocumentation/vacuum/MxidWraparound.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const MxidWraparoundTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const warning = config.settings["warning_threshold_pct"] as number;
+  const critical = config.settings["critical_threshold_pct"] as number;
+  return (
+    <p>
+      Detects when multixact ID space utilization on the server is more than{" "}
+      <code>{warning}%</code> and creates an issue with severity "warning".
+      Escalates to "critical" once utilization exceeds <code>{critical}%</code>.
+      Resolves once utilization falls below the threshold.
+    </p>
+  );
+};
+
+const MxidWraparoundGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { backendsUrl, serverVacuumFreezingUrl },
+}) => {
+  const Link = useSmartAnchor();
+  return (
+    <div>
+      <h4>Impact</h4>
+      <p>
+        Multixact ID space utilization is high and approaching to wraparound.
+      </p>
+      <p>
+        This is about a multixact ID, not about the normal transaction ID. A
+        multixact ID is an internal identifier used support row locking by
+        multiple transactions. Multixact IDs are commonly used with{" "}
+        <code>SELECT ... FOR UPDATE</code> query.
+      </p>
+      <p>
+        Postgres runs autovacuums regularly and this helps keeping multixact ID
+        space utilization low by freezing old multixact IDs. There is also the
+        anti-wraparound autovacuum specific for freezing, when the regular
+        autovacuums are either not freezing or simply not running. This
+        anti-wraparound autovacuum will be triggered when utilization (age)
+        exceeds the threshold, specified in{" "}
+        <code>autovacuum_multixact_freeze_max_age</code>.
+      </p>
+      <p>
+        In order to avoid wraparound failure, old multixact IDs must be freezed
+        by VACUUMs and this is not avoidable. The more old multixact IDs need to
+        be freezed, the more expensive VACUUM costs, potentially causes the
+        overall performance degradation or simply takes really long time to
+        finish. The anti-wraparound autovacuum holds{" "}
+        <code>SHARE UPDATE EXCLUSIVE</code> lock, which can block DDL
+        statements.
+      </p>
+      <p>
+        If multixact ID space utilization reaches to 99.85% (3M multixact left),
+        the system will shut down and refuse to start any new multixact. This is
```

Comment:
```suggestion
        If multixact ID space utilization reaches to 99.85% (3M multixact IDs left),
        the system will shut down. This is
```
I think that second part is sort of implicit, and you clarify it in the following sentence anyway.